content/sensu-go/5.20/operations/deploy-sensu/cluster-sensu.md

```diff
@@ -262,11 +262,44 @@ See [Secure Sensu][16] for information about cluster security.
 
 ## Use an external etcd cluster
 
-To use Sensu with an external etcd cluster, you must have etcd 3.3.2 or newer.
-To stand up an external etcd cluster, follow etcd's [clustering guide][2] using the same store configuration.
+Using Sensu with an external etcd cluster requires etcd 3.3.2 or newer but is not compatible with etcd 3.4.0 or newer.
+Follow etcd's [clustering guide][2] using the same store configuration to stand up an external etcd cluster.
 
-In this example, you will enable client-to-server and peer communication authentication [using self-signed TLS certificates][13].
-To start etcd for `backend-1` based on the [three-node configuration example][19]:
+To initialize a backend that uses etcd authentication, configure read and write access to the `/sensu.io/` key space for your users:
+
+{{< highlight shell >}}
+# Add the sensu user (you will receive an interactive prompt for the password)
+etcdctl user add sensu
+# Create a role
+etcdctl role add sensu_readwrite
+# Give that role permission to read and write under the /sensu.io/ keyspace
+etcdctl role grant-permission sensu_readwrite readwrite --from-key '/sensu.io/'
+# Grant the sensu user the role
+etcdctl user grant-role sensu sensu_readwrite
```

Comment:
@tarcinil your comment at https://github.com/sensu/sensu-docs/pull/2218#discussion_r435674198 indicates that we still need to add something more here, but I'm not sure exactly what. Can you clarify?

Reply:
I believe the comment is referring to the key at `/sensu.io/.initialized` which Sensu must be able to create in order to successfully initialize etcd database. AFAICT the current grant-permission would be sufficient to allow writing the `/sensu.io/.initialized` key.

Reply:
My tests are showing that @tarcinil's comment is correct -- an additional permission is needed for `sensu-backend init` to succeed. 

The init process tries to obtain a mutex lock, which attempts to create keys under `.initialized.lock`, so the permissions limited to `/sensu.io/` prefix are insufficient:

```
etcdserver: failed to apply request,took 16.179µs,request header:<ID:4554675860814564631 username:"sensu-agent" auth_revision:8 > txn:<compare:<target:CREATE key:".initialized.lock/3f3575904ee60d15" create_revision:0 > success:<request_put:<key:".initialized.lock/3f3575904ee60d15" lease:4554675860814564629 > request_range:<key:".initialized.lock/" range_end:".initialized.lock0" limit:1 sort_order:ASCEND sort_target:CREATE > > failure:<request_range:<key:".initialized.lock/3f3575904ee60d15" >  request_range:<key:".initialized.lock/" range_end:".initialized.lock0" limit:1 sort_order:ASCEND sort_target:CREATE > >>,resp ,err is auth: permission denied
```

I have tried granting permissions such as 
```
etcdctl role grant-permission sensu_readwrite readwrite --from-key '.initialized.lock'
```

But these too seem insufficient:

```
etcdserver: failed to apply request,took 9.573µs,request header:<ID:4554675860814564705 username:"sensu-agent" auth_revision:20 > txn:<compare:<target:CREATE key:".initialized.lock/3f3575904ee60d5f" create_revision:0 > success:<request_put:<key:".initialized.lock/3f3575904ee60d5f" lease:4554675860814564703 > request_range:<key:".initialized.lock/" range_end:".initialized.lock0" limit:1 sort_order:ASCEND sort_target:CREATE > > failure:<request_range:<key:".initialized.lock/3f3575904ee60d5f" >  request_range:<key:".initialized.lock/" range_end:".initialized.lock0" limit:1 sort_order:ASCEND sort_target:CREATE > >>,resp ,err is auth: permission denied
```

Using a slightly different key got a weird result:

```
etcdctl --user root:tacosrule role grant-permission sensu_readwrite readwrite --from-key '.initialized.lock/'
```

With this permission, sensu-backend init fails claiming permission denied, but the etcd server does not print any corresponding permission errors as it does with the other permissions tested. 🤔